app/logic/minor.py

```diff
@@ -15,7 +16,72 @@
 from app.models.individualRequirement import IndividualRequirement
 from app.models.certificationRequirement import CertificationRequirement
 from app.models.communityEngagementRequest import CommunityEngagementRequest
-
+from app.models.summerExperience import SummerExperience
+from app.models.otherExperience import OtherExperience
+import logging
+
+
+def createSummerExperience(username, form_data):
+    try:
+        logging.info(f"Form data received: {form_data}")
+
+        user = User.get(User.username == username)
+        content_area = ', '.join(form_data.getlist('contentArea'))  # Combine multiple content areas
+
+        # Determine the experience type
+        experience_type = form_data['experienceType']
+        if experience_type == 'Other':
+            other_experience_description = form_data.get('otherExperienceDescription', '')
+            if not other_experience_description:
+                raise ValueError("Other experience description is required.")
+            experience_type = other_experience_description
```

Comment:
Let's just assign `experience_type` above, and not use an additional variable.

Reply:
Fixed